storage/txpool/src/ops/tx_read.rs

```diff
@@ -34,3 +37,16 @@ pub fn get_transaction_verification_data(
         cached_verification_state: Mutex::new(cached_verification_state),
     })
 }
+
+/// Returns if the transaction with the given hash is in the stem pool.
+///
+/// This will return an err if the transaction is not in the pool.
```

Comment:
```suggestion
/// Returns `true` if the transaction with the given hash is in the stem pool.
///
/// # Errors
/// This will return an [`Err`] if the transaction is not in the pool.
```